lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -670,6 +670,564 @@ class AuthList
       success: (data) -> cb?(data)
       error: std_ajax_err => @$request_response_error.text gettext "Error changing user's permissions."
 
+class EmailSelectors
+  constructor: (@$container, @$section, params={}) ->
+    params = _.defaults params,
+      label : $container.data 'label'
+
+    template_html = $("#email-list-widget-template").html()
+    @$container.html Mustache.render template_html, params
+    @cur_column = 0
+    @$table = $("#queryTableBody")
+    @labelArray = ($container.data 'selections').split '<>'
```

Comment:
I'm not seeing `labelArray` referenced outside of this method; why are we assigning it as an instance variable instead of making it local?